backend/src/main/java/gov/cdc/usds/simplereport/api/model/pxp/PxpVerifyResponseV2.java

```diff
@@ -25,6 +30,12 @@ public PxpVerifyResponseV2(Person person, TestEvent testEvent) {
 
     this.testEventId = testEvent.getInternalId();
     this.result = testEvent.getResult();
+    Set<Result> allResults = testEvent.getResults();
+    results = new HashSet<>();
```

Comment:
For what I understand here we are just adding a new field results to the response that will contain an array instead of just one result (similar to the changes done for the test queue related endpoints). Just want to confirm that we will not run in any weird edge case while retrieving the results of the DB when the test is a correction.

Reply:
Just verified this - for removed events, we're still displaying the patient experience correctly and only returning a single result.